stdlib/typing.pyi

```diff
@@ -2,6 +2,7 @@ import collections  # Needed by aliases like DefaultDict, see mypy issue 2986
 import sys
 from abc import ABCMeta, abstractmethod
 from types import BuiltinFunctionType, CodeType, FrameType, FunctionType, MethodType, ModuleType, TracebackType
+from typing_extensions import Literal as _Literal
```

Comment:
Why is this necessary?

Reply:
Because Python 3.6 and 3.7 doesn't have `Literal`.

Reply:
I meant, why do we need to import it `as _Literal`.

Reply:
Because `typing` defines Literal itself and this would be a conflict. The alternative is to use a conditional import on < Python 3.8.

Reply:
Nevermind. Somehow I thought this was `re.pyi`.